full/src/main/java/apoc/load/LoadDirectoryHandler.java

```diff
@@ -146,6 +143,10 @@ private Runnable createListener(LoadDirectoryItem item) {
                     Thread.sleep(config.getInterval());
                 }
             } catch (Exception e) {
+                if (e instanceof InterruptedException) {
+                    return;
+                }
+                System.out.println("ERROR: " + ExceptionUtils.getStackTrace(e));
```

Comment:
what about `log` instead of `System.out.println`? (it should at least be `System.err.println`)

Reply:
that was just my debug message because with the test database there are no logs, I have to clean up the code, and remove it for sure!